pkg/syncer/cnsoperator/controller/cnsvolumemetadata/cnsvolumemetadata_controller.go

```diff
@@ -213,8 +213,8 @@ func (r *ReconcileCnsVolumeMetadata) Reconcile(ctx context.Context, request reco
 		if !r.updateCnsMetadata(ctx, instance, true) {
 			// Failed to update CNS
 			msg := fmt.Sprintf("ReconcileCnsVolumeMetadata: Failed to delete entry in CNS for instance "+
-				"with name %q and entity type %q in the guest cluster. Requeuing request.",
-				instance.Spec.EntityName, instance.Spec.EntityType)
+				"with name %q and entity type %q in the guest cluster %s. Requeuing request.",
```

Comment:
I prefer to use `%q` instead of `%s`. For any reason, if the string is empty, it will be easier to debug with `%q` format specifier.

Reply:
Isn't this redundant ?

By design, the first half of the instance Name always contain GCID.
Line : 191 is already printing the CnsVolumeMetadata instance Name.
For example:
```
ReconcileCnsVolumeMetadata: Received request for instance f0fed61e-d9a8-4146-aa91-e828b7e5f1c9-440a7605-9eb3-4375-8a16-d7892bcf3926  and type PERSISTENT_VOLUME"
```
In this case `f0fed61e-d9a8-4146-aa91-e828b7e5f1c9` is the GCID.

Reply:
@chethanv28 Yes.  I understand that by design  the first half of the instance Name always contain GCID. But it does not intuitive for someone who does not understand the internal design to figure out this volume metadata update request is come from which guest cluster. So I think we should better add this in the log.